tee-worker/ts-tests/identity.test.ts

```diff
@@ -123,67 +124,123 @@ const discordValidationData = <LitentryValidationData>{
 
 describeLitentry('Test Identity', (context) => {
     const aesKey = '0x22fc82db5b606998ad45099b7978b5b4f9dd4ea6017e57370ac56141caaabd12';
+
     const errorAseKey = '0xError';
     const errorCiphertext = '0xError';
     var signature_ethereum;
     var signature_substrate;
+    step('check user sidechain storage before create', async function () {
+        const encode = context.api.createType('LitentryIdentity', twitterIdentity).toHex();
```

Comment:
I think `encode` is not a good name - something like `identity_hex` would be better, or even `identity` when it's local. Same below